src/Pages/StatusPage/StatusPages/Components/StatusPagesTable/index.jsx

```
@@ -19,42 +19,46 @@ const StatusPagesTable = ({ data }) => {
 			id: "url",
 			content: "Public URL",
 			onClick: (e, row) => {
-				if (row.isPublished) {
-					e.stopPropagation();
-					const url =
-						row.type === "distributed"
-							? `/status/distributed/public/${row.url}`
-							: `/status/uptime/public/${row.url}`;
-					navigate(url);
-				}
+			  if (row.isPublished) {
```

ESLint

[error] Mixed spaces and tabs.

(no-mixed-spaces-and-tabs)